tests/testthat/test-summarize.R

```diff
@@ -0,0 +1,41 @@
+library("dplyr")
+library("tidylog")
+context("test-summarize")
+
+test_that("summarize", {
+    expect_message({
+        out <- summarize(mtcars, test = TRUE)
+    })
+    expect_equal(nrow(out), 1)
+
+    expect_message({
+        out <- mtcars %>%
+            group_by(cyl, carb) %>%
+            summarize(avg_mpg = mean(mpg))
+    })
+    expect_equal(nrow(out), 9)
+
+    expect_message({
+        out <- summarise_all(mtcars, max)
+    })
+    expect_equal(nrow(out), 1)
+
+    expect_message({
+        out <- summarise_at(mtcars, c("mpg", "hp"), mean)
+    })
+
+    expect_silent({
+        out <- dplyr::summarise(mtcars)
+    })
+})
+
+test_that("filter: scoped variants", {
```

Comment:
could you change this to `summarize` and move the tests for `summarise_all` and `summarise_at` here? (these are the scoped variants)

Reply:
I moved them into the scope, thanks for being so helpful!